automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -112,14 +112,14 @@ object Leonardo extends RestClient with LazyLogging {
       s"api/cluster${version.toUrlSegment}/${googleProject.value}/${clusterName.string}"
     }
 
-    def list()(implicit token: AuthToken): Seq[Cluster] = {
```

Comment:
I removed the old `list()` and `listIncludingDeleted()` because I think we should always be calling the per-project version.